webpack/react_app/components/RecentJobsCard/JobStatusIcon.js

```diff
@@ -22,9 +32,29 @@ const JobStatusIcon = ({ status, children, ...props }) => {
           <ExclamationCircleIcon {...props} /> {children}
         </span>
       );
+    case JOB_PLANNED_STATUS:
+      return (
+        <span className="job-planned">
+          <BuildIcon {...props} /> {children}
+        </span>
+      );
+    case JOB_RUNNING_STATUS:
+      return (
+        <span className="job-running">
+          <RunningIcon {...props} /> {children}
+        </span>
+      );
+    case JOB_CANCELLED_STATUS:
+      return (
+        <span className="job-cancelled">
+          <ExclamationTriangleIcon {...props} /> {children}
+        </span>
+      );
+    case JOB_AWAITING_STATUS:
+      return <span className="job-awaiting_start">{children}</span>;
     default:
       return (
-        <span className="job-info">
+        <span className="job-unknown">
```

Comment:
Only the icon should have the job* classname, so the text will always be black

Reply:
Also, can you import `webpack/react_app/components/RecentJobsCard/styles.scss` in the page, just for clearity on where the colours come from

Reply:
> can you import webpack/react_app/components/RecentJobsCard/styles.scss in the page

What do you please mean by that? `import './styles.scss';` in the `JobStatusIcon.js` isn't enough?

Reply:
I missed that, nvm :)